var/spack/repos/builtin/packages/py-vmd-python/package.py

```diff
@@ -35,6 +35,7 @@ class PyVmdPython(PythonPackage):
 
     depends_on('python@2.7:2.8')
     depends_on('py-numpy')
+    depends_on('py-setuptools', type='run')
```

Comment:
are you sure that it is a `run` but not a `build` dependency?

Reply:
The `setup.py` doesn't reference setuptools but I do see this:
```console
$ grep -R pkg_resources
vmd/vmd_src/__init__.py:from pkg_resources import resource_filename
```
so I think this looks right.